docs/README.md

```diff
@@ -75,6 +108,229 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+    default-v1-21-0   True    Healthy   False    v1.21.0   114s
+    ```
+Note: `IstioRevision` name is in the format `<Istio resource name>-<version>`.
+
+5. Create `bookinfo` namespace and label it with the revision name
+
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo istio.io/rev=default-v1-21-0
+    ```
+
+6. Deploy bookinfo application
+
+    ```bash
+    kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml
+    ```
+
+7. Confirm proxy sidecar injection version match the control plane version
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should match the control plane version.
+
+8. Update the control plane to a new version
+
+    ```bash
+    kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.21.2"}}'
+    ```
+
+9. Verify the `Istio` and `IstioRevision` resources. There will be a new revision created with the new version.
+
+    ```console
+    $ kubectl get istio -n istio-system
+      NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   VERSION   AGE
+      default   2           2       1        Healthy           v1.21.2   23m
+    $ kubectl get istiorevision -n istio-system
+      NAME              READY   STATUS    IN USE   VERSION   AGE
+      default-v1-21-0   True    Healthy   True     v1.21.0   27m
+      default-v1-21-2   True    Healthy   False    v1.21.2   4m45s
+    ```
+
+10. Confirm there are two control plane pods running, one for each revision
+
+    ```console
+    $ kubectl get pods -n istio-system
+      NAME                                      READY   STATUS    RESTARTS   AGE
+      istiod-default-v1-21-0-69d6df7f9c-grm24   1/1     Running   0          28m
+      istiod-default-v1-21-2-7c4f4674c5-4g7n7   1/1     Running   0          6m9s
+    ```
+
+11. Confirm the proxy sidecar version remains the same:
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should still match the old control plane version.
+
+12. Change the label of the `bookinfo` namespace to use the new revision
+
+    ```bash
+    kubectl label namespace bookinfo istio.io/rev=default-v1-21-2 --overwrite
+    ```
+The injected version by the control plane will not change immediately. It will only be changed after the pods are restarted, this will trigger the sidecar injection with the new version.
+
+13. Delete all the pods in the `bookinfo` namespace
+
+    ```bash
+    kubectl rollout restart deployment -n bookinfo
+    ```
+
+14. Confirm the new version is used in the sidecars
+
+    ```bash
+    istioctl proxy-status 
+    ```
+The column `VERSION` should match the updated control plane version.
+
+15. Confirm the old control plane deletion
+
+The old control plane will be deleted after the grace period specified in the `Istio` resource `Istio.spec.updateStrategy.inactiveRevisionDeletionGracePeriodSeconds`.
```

Comment:
nit: Since we are kind of repeating similar message even in Line 327, we can probably club number 15 and 16 together and show both the outputs to avoid the duplication.